Makefile

```diff
@@ -203,11 +205,11 @@ out/doc/api/assets/%: doc/api_assets/% out/doc/api/assets/
 out/doc/%: doc/%
 	cp -r $< $@
 
-out/doc/api/%.json: doc/api/%.markdown $(NODE_EXE)
+out/doc/api/%.json: doc/api/%.markdown
 	$(NODE) tools/doc/generate.js --format=json $< > $@
 
-out/doc/api/%.html: doc/api/%.markdown $(NODE_EXE)
-	$(NODE) tools/doc/generate.js --format=html --template=doc/template.html $< > $@
+out/doc/api/%.html: doc/api/%.markdown
+	$(NODE) tools/doc/generate.js --node-version=$(RAWVER) --format=html --template=doc/template.html $< > $@
```

Comment:
@rvagg Ah, thank you! I pushed a new commit that corrects that. I realized it was a mistake anyway since I overlooked that `process.version` is prefixed with `v` and `RAWVER` isn't.